lib/datadog/core/telemetry/client.rb

```diff
@@ -65,6 +67,13 @@ def integrations_change!
           @emitter.request(:'app-integrations-change')
         end
 
+        # Report configuration changes caused by Dynamic Configuration.
+        def dynamic_configuration_change!(changes)
```

Comment:
Looks like there is a mismatch in the naming. `dynamic_configuration` and `remote_config`

Reply:
Renamed it to match the telemetry event.